index.js

```diff
@@ -1,24 +1,21 @@
 var sep = require('path').sep
-var last = require('lodash').last
 var toString = require('mdast-util-to-string')
 
 module.exports = checkTitle
 
-function checkTitle (remark, opts) {
+function checkTitle (opts) {
+  var title = (opts || {}).title
   return function checkTitleTransformer (root, file) {
+    var filePath = (file.dirname === '.' ? file.cwd : file.dirname) || title || process.cwd();
+    var dirnames = filePath.split(sep)
+    var folder = dirnames[dirnames.length - 1].toLowerCase()
```

Comment:
Oh I especially used `folder` as I saw it in the README.
One thing I found odd btw, is that `title` is not used if there’s a directory name.
I think it should be the other way around: use the title first, fall back to the directory otherwise.